### ISSUE TITLE
On selecting any menu item from 'Email' drop down, default value 'Email' is showing

### ISSUE THREAD

**Issue by ayushjai19** [microsoft/WinUI-Gallery#2082]
### Which version of the app?

WinUI 3 Gallery

### Description

Environment Details:
OS Version: 24H2 (OS Build 26100.4315)
App: WinUI 3 Gallery
App Version: 2.6.0.0 (SDK 1.7)

Repro Steps:
Launch the WinUI 3 Gallery app.
Navigate to search box using tab key.
Type 'DropDownButton' in search box and hit enter.
Navigate to 'Email' dropdown button and select any  menu item from that drop down and observe that drop down.
Actual Result:
On selecting any menu item from 'Email' drop down, default value 'Email' is showing
 
Note: Same issue is observed for 'Email Icon' drop down present under 'DropDownButton' with icons group

Expected Result:
Once after selecting any menu item from 'Email' dropdown, selecting menu item should be shown in the drop down instead of default value

Xaml page code link: https://github.com/microsoft/WinUI-Gallery/blob/main/WinUIGallery/Samples/ControlPages/DropDownButtonPage.xaml

### Windows version

Windows 11 24H2 (26100)

**Comment by ayushjai19** [comment]
This is not a bug, it is by design. DropDownButton is used to show actions for a task not to set the content of button.
